.github/workflows/release.yml

```diff
@@ -20,6 +20,11 @@ jobs:
       -
         name: Set up Go
         uses: actions/setup-go@v5
+
+      -
+        name: Golangci-lint
+        uses: golangci/golangci-lint-action@v6.0.1
```

Comment:
I'm noticing I added this one on the release process, so only when something is tagged